e2e/update/src/generic-tests.ts

```diff
@@ -0,0 +1,50 @@
+import { browser } from 'protractor';
+import { BaseDevUpdateTestPage } from './update.po';
+
+const page = new BaseDevUpdateTestPage();
```

Comment:
Can you try passing in the page to the functions instead of creating a new page?  So calling a test might look like:

`testPageLoad(profilePage, '/sitereg/profile')`

That way you can also just call "navigateTo()" like you would normally instead of "navigateToURL()"